Makefile

```diff
@@ -17,4 +17,8 @@ down:
 	- docker-compose -f docker-compose.yml -f docker-compose.local.yml down
 
 test:
+	- rm -rf source/storage/database.sqlite
+	- touch source/storage/database.sqlite
+	- docker exec -it squabble_web_1 php artisan migrate
 	- docker exec -it squabble_web_1 phpunit
+	- vendor/bin/behat
```

Comment:
Hmm let's talk about this -- my goal is to run integration tests against the container itself, since that's how it will be deployed in production. That way the behat baseline would also catch networking errors, etc that came from container configuration changes.. E.g. spinning up the server and then testing it via behat.